libyara/modules/pe.c

```diff
@@ -1304,7 +1304,7 @@ void pe_parse_certificates(
           // space to hold it, and use i2d_ASN1_INTEGER() one last time to
           // hold it in the allocated buffer.
 
-          unsigned char* serial_der = yr_malloc(bytes);
+          unsigned char* serial_der = (unsigned char*) yr_malloc(bytes);
```

Comment:
Actually, the VS projects provided have the "/TP" switch enabled (Configuration -> C/C++ -> Advanced -> Compile As), which forces the C files to be compiled as C++ files. So the error is:
..\..\..\libyara\modules\pe.c(1307): error C2440: 'initializing': cannot convert from 'void *' to 'unsigned char *'

If the files are compiled as a C, it compiles fine.

I'm using VS2015 with Update 3.